mne/epochs.py

```diff
@@ -1810,6 +1810,56 @@ def save(self, fname, split_size='2GB', fmt='single', overwrite=False,
             this_epochs.event_id = self.event_id
             _save_split(this_epochs, fname, part_idx, n_parts, fmt)
 
+    def export(self, fname, fmt="auto"):
+        """Export Epochs to EEGLAB's .set format.
+
+        Parameters
+        ----------
+        fname : str
+            Name of the export file.
+        fmt : str
+            Format of the export. Defaults to auto, which will infer
+            format from fname's extension
+
+        Notes
+        -----
+        Channel locations are expanded to the full EEGLAB format
+        For more details see eeglabio.utils.cart_to_eeglab
```

Comment:
```suggestion
        Channel locations are expanded to the full EEGLAB format.
        For more details see ``eeglabio.utils.cart_to_eeglab``.
```

Would be ideal if we could cross-link to eeglabio docs via intersphinx; do you know how to set that up @jackz314?

Reply:
I haven't done it before, I can look into it but some help would be appreciated.

Reply:
Here is the basics: https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html

For configuring our end, see `doc/conf.py` around line 129 or so

For the eeglabio end, you'll need to create the `objects.inv` file, which means the eeglabio docs need to also be built with sphinx (from the URL above: "Each Sphinx HTML build creates a file named `objects.inv` that contains a mapping from object names to URIs relative to the HTML set’s root"). Once that exists someplace on the internet (i.e., once eeglabio has a website for its docs) you add the package name and URL to our conf.py and it should "just work".

Probably easiest is to try setting up intersphinx in the eeglabio docs `conf.py` file and try linking in those docs to some function within MNE-Python or NumPy.  That should be easier to test/iterate.  Once that works, it should work for us to link to eeglabio docs too.

Reply:
I published the docs at [eeglabio.readthedocs.io](https://eeglabio.readthedocs.io/). Also changed `doc/conf.py`.